src/pages/dashboard/components/total-sales.tsx

```diff
@@ -0,0 +1,221 @@
+"use client"
```

Comment:
You don't need to add `"use client"` in this project because this project is not NextJS based.
Can you remove this please?